tests/color/run.sh

```diff
@@ -1 +1 @@
-./bin/test_color_9.0_x86_64 --graph-type=market --graph-file=/media/minh/UStorage/git/color/gunrock/dataset/small/test_cc.mtx --seed=123 --usr_iter=1
+./bin/test_color_10.0_x86_64 --graph-type=market --graph-file=/home/minh/git/gunrock/dataset/small/test_cc.mtx --JPL=true --seed=123 --usr_iter=5
```

Comment:
Uses a hard reference to your folder, the small dataset exists under gunrock, so maybe do `../../dataset/small/test_cc.mtx`?